projects/core/src/cms/store/reducers/page.reducer.ts

```diff
@@ -1,11 +1,6 @@
 import * as fromPageData from '../actions/page.action';
-import { Page } from '../../models/page.model';
-
-export interface PageState {
-  entities: { [context: string]: Page };
-  count: number;
-  latestPageKey: string;
-}
+import { Page } from '../../model/page.model';
+import { PageState } from '../cms-state';
 
```

Comment:
Please move selectors at the end of reducers to selectors (where memoized selectors are defined)